app/javascript/react/ansibleCatalog/ansiblePlaybookFields.jsx

```diff
@@ -0,0 +1,344 @@
+                  }
+                }}
+                meta={meta}
+              />}
+          />
+          {
+            formFields.map(field => renderFormField(field, formValues))
+          }
+          <Field
+            name={`${prefix}_cloud_type`}
+            label={__('Cloud Type')}
+            render={({ input, meta }) =>
+              <FinalFormSelect
+                inputColumnSize={8}
+                labelColumnSize={3}
+                name={`${prefix}_cloud_type`}
+                options={dropdowns.cloudTypes || []}
+                label={__('Cloud Type')}
+                placeholder={DEFAULT_PLACEHOLDER}
+                input={{
+                  ...input, onChange: value => {
+                    if (value) {
+                      this.updateCloudCredentialsList(value);
+                    }
+                    input.onChange(value)
+                  }
+                }}
+                meta={meta}
+              />}
+          />
+          {
+            (formValues[`${prefix}_cloud_type`] !== '') &&
+               <Field
+                inputColumnSize={8}
+                labelColumnSize={3}
+                name={`${prefix}_cloud_credential_id`}
+                label={__('Cloud Credentials')}
+                component={FinalFormSelect}
+                options={dropdowns[`${prefix}_cloudCredentials`] || []}
+                placeholder={DEFAULT_PLACEHOLDER}
+              />
+          }
+        </div>
+        <div className="col-md-12 col-lg-6">
+          <div className="form-group">
+            <label className="control-label col-xs-2">
+              {__('Variables & Default Values')}
+            </label>
+            <div className="col-xs-8">
+              <div>
+                <Button
+                  type="button"
+                  bsStyle="primary"
+                  onClick={() => {
+                    this.addExtraVarToState();
+                    addExtraVars(extraVarsField, undefined)}
+                }>
+                  {__('Add')}
+                </Button>
+              </div>
+              {
+                formValues[extraVarsField].length ? (
+                  <table className="table table-bordered table-striped">
+                    <thead>
+                      <tr>
+                        <th>{__('Variable')}</th>
+                        <th>{__('Default')}</th>
+                        <th>{__('Action')}</th>
+                      </tr>
+                    </thead>
+                    <tbody>
+                      <FieldArray name={extraVarsField}>
+                        {({ fields }) =>
+                          fields.map((name, index) => { 
+                            const isEditing = extraVarsEditingState[index].editing;
+                            const trashIcon = `pficon ${(isEditing? 'pficon-close': 'pficon-delete' )}`;
+                            const editIcon = `pficon ${(isEditing? 'pficon-save': 'pficon-edit' )}`;
```

Comment:
You are using these icons only once. Use them directly it should also save some lines.